IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/ElementHandler.cs

```diff
@@ -166,12 +160,14 @@ public IWebElement GetWebElement()
                 typeof(NoSuchElementException)
                 );
 
+#pragma warning disable CS8603 // Possible null reference return. Needed for proper WebDriverWait behavior
```

Comment:
Why this disable? Does a `null` return from the delegate not get returned from the `Until` method?

Reply:
It does not. It's a quirk in how Selenium works. Basically, if the Until method gets a null response when T is a nullable non-bool object, it retries the delegate. If the timeout is reached it throws a WebDriverTimeout exception.

Reply:
@Keboo Any issues with the warning disable and explanation? Otherwise I'm thinking of buttoning up the changes to this and pushing out v2.0.0.